### ISSUE TITLE
Understanding 3D CNN

### ISSUE THREAD

**Issue by anujshah1003** [pydatabangalore/talks#23]
# Title
> Understanding 3D CNN

## Description
> What happens when we extend 2D CNN to temporal domain, This talk is about understanding the 3D CNN architecture and how the feature map changes when we traverse through each layer. I will  be using Keras and pictorial slides for explanation

## Duration
- 45 min 

## Audience
> Understanding of Neural Networks is the basic Prerequisite. It will be a big advantage if audience already knows the basic architecture of 2D CNN. This would be for intermediate audience.

## Outline
> I will update the detailed outline after few days

## Additional notes
> Anuj Shah
    Linkedin - https://www.linkedin.com/in/anuj-shah-759149117/
    Github - https://github.com/anujshah1003
    Youtube - https://www.youtube.com/channel/UCjAOM-s-f2YfeLNGGZuY-Bg
   Medium - 
---

- [ ] Don't record this talk.
> Check this if you don't want your talk to be recorded.

**Comment by vinayak-mehta** [comment]
@anujshah1003 Can you update the detailed outline?

**Comment by TrigonaMinima** [comment]
@anujshah1003 could you update the outline? We are finalizing the talks for the Jan meetup.

**Comment by TrigonaMinima** [comment]
@anujshah1003 closing the issue. Will open the issue once you have update the outline.
